### PR TITLE
added confounders from bridge2ai and some demographics fixes

### PR DESCRIPTION
**Pull request by sooyounga**
added confounders qns from bridge2ai, based on the current version as of March 17, 2023.
added some fixes for demographics qns 

ENT history is divided into multiple categorical qns with slight deviations from the qn

### CODE REVIEW

File: activities/demographics_and_background_information_v1/items/ent_history [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        "inputType": "radio"
+    },
+    "responseOptions": {
+        "valueType": "xsd:int",
+        "multipleChoice": true,
+        "choices": [
+            {
+                "name": {
+                    "en": "Chronic ear infection",
+                    "es": ""
```
[the remark this answers is not in the document]
Comment (satra) [reply activities/demographics_and_background_information_v1/items/ent_history:23]: also we can ask at the meeting today, if a spanish speaker/clinician could help us translate this.

**Comment by satra** [comment]
@ibevers - can you take over this branch? this is also relevant to the mood protocol. perhaps branch of this branch and update as needed and also push to `enh/rc4`

**Comment by satra** [comment]
this is the demographic + confounders section of the app (probably one of the most length and time consuming). hence this PR and the conversion of data elements from the b2ai redcap should help curate that list. we can sync up next week on this.

**Comment by yibeichan** [comment]
hi hi @ibevers @satra what's the final decision on this PR?

**Pull request closed without merging**